src/main/platform_specific/raspberrypi/debug.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include "debug.h"
+
+#define TRUE 1
+#define FALSE 0
+
+static uint8_t mext2_log_level = WARNING;
+static uint8_t mext2_is_log_file_initialized = FALSE;
+static FILE* mext2_log_file = NULL;
+
```

Comment:
can we all static keywords replace with STATIC and then add to some header file:
`#define STATIC static`
when there will be function that uses global variable and we would like to test this function only change that will be needed is 
`#define STATIC `
in test directory 
a po polskiemu: chodzi mi o to że funkcja testowa musi mieć dostęp do globalnych zmiennych statycznych, przedefiniowanie STATIC na nic pozwoli zmienić wartości tych zmiennych tak żeby nam pasowały do testu.
Żeby dało się przedefiniować taką zmienną potrzebujemy w każdym z katalogów z testami dodatkowego folderu ,,stubs", który będzie zawierał wszystkie pliki nagłówkowe jakie 'includuje' nasz testowany moduł. Potem w pliku testowym trzeba podać ścieżki do tych nagłówków. Chodzi o to żeby crc_test.c funkcję crc7() pobierał ze źródeł ale nagłówki z naszego lokalnego folderu stubs. Z tych stubów będziemy też tworzyć potem mocki więc jest bardzo potrzebny. W takiej replice nagłówka można wtedy zmieć #define STATIC static na #define STATIC